code/modules/projectiles/updated_projectiles/magazines/revolvers.dm

```diff
@@ -100,7 +110,9 @@
 /obj/item/ammo_magazine/internal/revolver/small
 	default_ammo = /datum/ammo/bullet/revolver/small
 	caliber = ".357"
-	gun_type = /obj/item/weapon/gun/revolver/small
+	gun_type = list(

```

Comment:
The current system doesn't work for lists. This needs a refactor to work.

Reply:
This hasn't been fixed. Please do not resolve conversations on your own without addressing the issues.